CMakeLists.txt

```diff
@@ -96,11 +96,26 @@ if (OTBR_BACKBONE_ROUTER)
     target_compile_definitions(otbr-config INTERFACE
             OTBR_ENABLE_BACKBONE_ROUTER=1
     )
+    set(OTBR_BACKBONE_ROUTER_DUA_NDPROXYING ON CACHE BOOL "Backbone Router enables DUA ND Proxying by default")
     set(OT_THREAD_VERSION 1.2 CACHE STRING "Backbone Router requires Thread 1.2 or higher" FORCE)
     set(OT_BACKBONE_ROUTER ON CACHE BOOL "Enable Backbone Router feature in OpenThread" FORCE)
     set(OT_SERVICE ON CACHE BOOL "Backbone Router requires Thread network service" FORCE)
 endif()
 
+
+option(OTBR_BACKBONE_ROUTER_DUA_NDPROXYING "Enable Backbone Router DUA ND Proxying" OFF)
```

Comment:
It seems the new `OTBR_BACKBONE_ROUTER_DUA_NDPROXYING` is equivalent with `OTBR_ENABLE_BACKBONE_ROUTER` from usage's perspective. May keep using  `OTBR_ENABLE_BACKBONE_ROUTER`?

Reply:
Agree that we can keep only `OTBR_ENABLE_BACKBONE_ROUTER`.

Reply:
fixed